renderer/components/editor/index.js

```diff
@@ -37,9 +37,26 @@ export default class Editor extends React.Component {
         <div className="title-bar">
           <div className="title-bar-container">
             <div className="traffic-lights">
```

Comment:
The traffic lights should be extracted into a component.

Reply:
For code clarity? Because I don't think we use them anywhere else

Reply:
Yes, for code clarity. They're totally self-contained.